### DISCUSSION TITLE
"dep deploy" does not exist, offers an option list

### DISCUSSION THREAD

**Discussion by rowild** [deployphp/deployer#2306] [Help needed]
- Deployer version: 6.8.0
- Deployment OS: Mac OS 10.13.6

As a beginner, I am working through the "Getting startet" chapter. At "dep deploy" I get this error:

```sh
Command "deploy" is not defined.  
                                    
  Did you mean one of these?        
      deploy:clear_paths            
      deploy:copy_dirs              
      deploy:lock                   
      deploy:prepare                
      deploy:release                
      deploy:shared                 
      deploy:symlink                
      deploy:unlock                 
      deploy:update_code            
      deploy:vendors                
      deploy:writable   
```

Why?

**Comment by antonmedv** [comment]
Because deploy task not defined.

**Comment by rowild** [comment]
Shouldn#t that be mentioned in a "Getting started" guide?

**Comment by antonmedv** [comment]
Yes I think so. Also i have plans to rewrite docs from scratch. Also it’s possible to add deploy task to common recipe as well.

**Comment by rowild** [comment]
Thank you! It would be okay for to close this issue, if you are all right with that.
Have a good day!
